Documentation/api-reference.md

```diff
@@ -23,11 +23,11 @@ val mediaExtensionVersion = Media.extensionVersion()
 
 ### EXTENSION
 
-Represents a reference to AssuranceExtension.class that can be used to register with MobileCore via its registerExtensions api.
+Represents a reference to MediaExtension.class that can be used to register with MobileCore via its registerExtensions api.
 
 #### Syntax
 ```java
-public static final Class<? extends Extension> EXTENSION = AssuranceExtension.class;
+public static final Class<? extends Extension> EXTENSION = MediaExtension.class;
```

Comment:
Also need to change `Assurance.EXTENSION` to `Media.EXTENSION` in the examples below on lines 37 and 44.